src/App.tsx

```diff
@@ -1,9 +1,24 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const sortedMovies = () => {
```

Comment:
you can refactor this to store result in variable, e.g.:

```
const sortedMovies = moviesFromServer.filter(movie => {
      const title = movie.title.toLowerCase();
      const description = movie.description.toLowerCase();

      return title.includes(query) || description.includes(query);
    });
```

then you avoid using function call is MoviesList props so this increases performance and avoid unexpected re-renders calls:

`<MoviesList movies={sortedMovies} />`